build.sbt

```diff
@@ -125,6 +128,7 @@ lazy val client = (project in file("clients/java"))
       val _ = openApiGenerate.value
     }
   )
+  .settings(javaOnlyReleaseSettings: _*)
```

Comment:
you dont need this. you can just put the javaOnlyReleaseSettings without expansion in the previous settings block. see https://github.com/delta-io/delta/blob/master/build.sbt#L589

Reply:
Done